### PR TITLE
Resolves #2436

### LINKED ISSUE #2436: Bad Links On Doc page for AppvClient
Looks like all the module links @ https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/mdop/appvclient/appvclient.md are bad now, as in 404.

### PR DESCRIPTION
**Pull request by velkovb**
Fix broken links
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2436

### CODE REVIEW

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@velkovb - please relook all the added links because they are not formatted correctly, for example, "./get-appvpublishingserver.md./remove-appvpublishingserver.md./set-appvpublishingserver.md./sync-appvpublishingserver.md". Please look at the rich diff view for guidance if necessary. Many thanks.

**Comment by velkovb** [comment]
Seems like something with my regex went bad. I will revisit.

File: docset/mdop/appvclient/add-appvclientpackage.md [hunk 1]
```diff
@@ -113,11 +113,12 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
-
+./mount-appvclientpackage.md
 [Stop-AppvClientPackage](./Stop-AppvClientPackage.md)
-
+./publish-appvclientpackage.md
 [Unpublish-AppvClientPackage](./Unpublish-AppvClientPackage.md)
+./publish-appvclientpackage.md
 
-
+./remove-appvclientpackage.md./repair-appvclientpackage.md./set-appvclientpackage.md./stop-appvclientpackage.md./unpublish-appvclientpackage.md
```
Comment (get-itips) [inline docset/mdop/appvclient/add-appvclientpackage.md:124]: is there a reason why these are unformatted links?

Comment (velkovb) [reply docset/mdop/appvclient/add-appvclientpackage.md:124]: Its messed up. I will fix it.

**Review by dariomws — COMMENTED** [review]
I'll review again after the links are fixed. Thanks!

**Comment by velkovb** [comment]
@dariomws @JohanFreelancer9 @get-itips Should be OK now. Not really sure how I achieved the previous state as it is the same regex :)

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**